lib/loader/index.js

```diff
@@ -126,27 +130,37 @@ function postInstantiate(baseModule, instance) {
   }
 
   /** Gets the runtime alignment of a collection's values or keys. */
-  function getAlign(which, info) {
-    return 31 - Math.clz32((info / which) & 31); // -1 if none
+  function getAlign(info) {
+    return 31 - Math.clz32((info >>> VAL_ALIGN_OFFSET) & 31); // -1 if none
```

Comment:
Just noticed: Removing the `which` argument will prevent us from using this function to obtain the key alignment of maps. We don't use that currently, but a comment to easily re-add this if necessary would help.

Reply:
I'll revert back it and rename modified `getAlign` to `getAlignValue`